app/views/site/update_user_annotations.js.erb

```diff
@@ -0,0 +1,11 @@
+//reload the menu after saving a user annotaion
+closeModalSpinner('#save-modal-spinner', '#save-modal', function () {
+    $("#annotation").html("<%= escape_javascript(select_tag :annotation, grouped_options_for_select(@cluster_annotations, params[:annotation]), class: 'form-control' )%>");
+    $('#notices-target').html("<%= escape_javascript(render partial: '/layouts/notices', locals: {notice: @notice, alert: @alert}) %>");
+    <% if @user_annotation.errors.any? %>
+
```

Comment:
You ideally should be checking to see where the errors are occurring rather than assuming they're in the name.